### DISCUSSION TITLE
Bug in Version 1.0.187

### DISCUSSION THREAD

**Discussion by myracullu** [Eugeny/tabby#7738] [General]
In profile and connections to open o host the window of vault apear , but if put esc the vault windows disapear and the host can be edited whitout password vault... to remove host same.

**Comment by Eugeny** [comment]
If you don't enable full config encryption, the vault is only used to store SSH passwords, that's what the prompt is for.

**Reply by myracullu** [reply]
i whould like that vault just password in tabby login , when duplicate connection ssh dont , and when edit the host too. It is possible ? How to ?

**Reply by Eugeny** [reply]
You can just ignore the vault password prompt then - if you cancel it, Tabby simply won't show whether there's a saved password for a given profile.
